chain/db/nanopay.go

```diff
@@ -0,0 +1,44 @@
+package db
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/nknorg/nkn/common"
+	"github.com/nknorg/nkn/common/serialization"
+)
+
+type nanoPay struct {
+	balance common.Fixed64
+	limit   uint32
+}
+
+func (np *nanoPay) Serialize(w io.Writer) error {
+	err := np.balance.Serialize(w)
+	if err != nil {
+		return fmt.Errorf("account nanoPay Serialize error: %v", err)
+	}
+
+	err = serialization.WriteUint32(w, np.limit)
+	if err != nil {
+		return fmt.Errorf("account nanoPay Serialize error: %v", err)
+	}
+
+	return nil
+}
+
+func (np *nanoPay) Deserialize(r io.Reader) error {
+	var balance common.Fixed64
+	err := balance.Deserialize(r)
```

Comment:
Is there a reason not to use `np.balance.Deserialize(r)` here?

Reply:
I guess not, fixed